tests/CronofyTest.php

```diff
@@ -307,6 +307,39 @@ public function testDeleteEvent()
         $this->assertNotNull($actual);
     }
 
+    public function testBulkDeleteEvents()
+    {
+        $params = ["calendar_ids" => ["cal_123"], "delete_all" => true];
+
+        $http = $this->createMock(HttpRequest::class);
+        $http->expects($this->once())
+            ->method('httpDelete')
+            ->with(
+                $this->equalTo('https://api.cronofy.com/v1/events'),
+                $this->equalTo($params),
+                $this->equalTo([
+                    'Authorization: Bearer accessToken',
+                    'Host: api.cronofy.com',
+                    'Content-Type: application/json; charset=utf-8',
+                ])
+            )
+            ->will($this->returnValue(["{'foo': 'bar'}", 200]));
+
+        $cronofy = new Cronofy([
+            "client_id" => "clientId",
+            "client_secret" => "clientSecret",
+            "access_token" => "accessToken",
+            "refresh_token" => "refreshToken",
+            "http_client" => $http,
+        ]);
+
+        $actual = $cronofy->bulkDeleteEvents([
+            "calendar_ids" => ["cal_123"],
+            "delete_all" => true
+        ]);
```

Comment:
The real API would reject this request as only one or the other parameter should be passed